files/zh-cn/web/css/css_colors/applying_color/index.md

```diff
@@ -206,42 +214,206 @@ th {
 
 {{EmbedLiveSample("HSL 函数表示法", 300, 260)}}
 
-> **备注：** Note that when you omit the hue's unit, it's assumed to be in degrees (`deg`).
+> **备注：** 如果省略了色相的单位，则假定其单位为度（`deg`）。
+
+#### HWB 函数表示法
+
+与上面的 HSL 函数表示法类似，[hwb()](/zh-CN/docs/Web/CSS/color_value/hwb) 函数使用相同的色相值。不过，指定的白度和黑度值不是亮度和饱和度，而是百分比。数值之间**不用**逗号分隔，还可以包含一个可选的 alpha 值（必须在前面加上斜线 `/`）。
+
+下面是一些用 HWB 符号表示的颜色示例：
+
+```css
+/* 这些示例都指定了深浅不一的青绿色。 */
+hwb(90 10% 10%)
+hwb(90 10% 10%)
+hwb(90 50% 10%)
+hwb(90deg 10% 10%)
+hwb(1.5708rad 60% 0%)
+hwb(.25turn 0% 40%)
+
+/* 同样的青绿色，这次包含了透明度值 */
+hwb(90 10% 10% / 0.5)
+hwb(90 10% 10% / 50%)
+```
+
+### 颜色空间和颜色模型
+
+十六进制、具名颜色和 `rgb()` 函数都使用 [RGB](/zh-CN/docs/Glossary/RGB) 模型，并与 sRGB（`srgb`）色彩空间相关联。你可能会看到“色彩模型”和“色彩空间”交替使用，但这两者之间还是有区别的，值得注意：
+
+- 色彩模型是用数值表示色彩的数学模型。色彩模型描述了如何在色彩空间内创建可用的色彩。
+- 色彩空间是对色彩进行分组的系统，因此对任何给定色彩的描述都是一致的。如果在不同的色彩空间之间转换一种颜色，它看起来会完全相同。
+
+我们上面使用的 `hsl()` 和 `hwb()` 函数使用的是 sRGB 色彩空间，并且都使用*柱形*模型；这就是为什么 `<hue>` 角度可以让你像在[色轮](/zh-CN/docs/Glossary/Color_wheel)上一样控制颜色的属性。让我们来看看其他可用的色彩空间以及使用它们的原因。
+
+#### CIELAB 和 Oklab 颜色空间
+
+CIELAB 和 Oklab 色彩空间基于人类视觉实验，代表了人类所能看到的全部色彩范围。这些模型的主要目的在于它们是统一的，因此色彩空间中任意两点之间的给定距离在观察者看来应该是同样不同的。
+
+对于 CIELAB 色彩空间，可以使用 `lab()` 和 `lch()` 函数。`lch()` 函数使用亮度（L）、色度（C）和色调（H），而 Lab 模型则沿色彩空间中的“a”轴和“b”轴（矩形坐标）使用亮度（L）、红/绿度和黄/蓝度。
+
+下面的示例显示了在 `lab()` 和 `lch()` 函数中更改分量值的效果，其中每一行都修改了一个分量。第一行显示的是对 `lch()` 亮度值的更改，第二行更改的是 `lch()` 色调分量。第三行更改了 `lab()` 的“b”轴，因此我们获得了与 `lch()` 不同的效果，因为我们是在逐渐增加颜色中的黄色，而不是在整个色轮中循环：
+
+```css hidden live-sample___lch-colors
```

Comment:
![image](https://github.com/mdn/translated-content/assets/40999116/1f0e6d65-a07a-43b0-8b72-bbcf791448c9)
It may be live sample's id? Without that the 1st param of the EmbedLiveSample call below({{embedlivesample("lch-colors", '100', '150') }}) would be longer here.

Reply:
Thanks for the note @jasonren0403. I found the functions in yari: https://github.com/mdn/yari/blob/f91600bfcb262a4b63d1fb2d143a3428d712cf3e/client/src/document/code/playground.ts#L240

So we does need the language tag (in other words: class name)